library/include/ck/library/reference_tensor_operation/cpu/reference_conv_backward_weight.hpp

```diff
@@ -70,18 +70,26 @@ struct ReferenceConvBwdWeight : public device::BaseOperator
             constexpr auto I1 = Number<1>{};
             auto f_kcyx       = [&](auto k, auto c, auto y, auto x) {
                 float v_acc = 0;
-                for(int n = 0; n < arg.out_n_k_ho_wo_.mDesc.GetLengths()[0]; ++n)
+                for(int n = 0; n < ck::type_convert<int>(arg.out_n_k_ho_wo_.mDesc.GetLengths()[0]);
```

Comment:
Consider `size_t i`

Reply:
There would be hidden size_t -> int then when indexing tensors in the most inner loop.

Reply:
again, hosttensordescriptor can take size_t as indices.

Reply:
so no convert is needed.

Reply:
Addressed by [594a4c2](https://github.com/ROCmSoftwarePlatform/composable_kernel/pull/212/commits/594a4c22d129e4eac6930355ea180a0e156735e5)